recipes/glimpse-bio/build.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
 
+export COMMIT_VERS="${GIT_FULL_HASH}"
+export COMMIT_DATE="$(date -Idate -u)"
+
+for subdir in chunk concordance split_reference phase ligate
 
-for subdir in chunk concordance ligate phase sample snparray
 do
     pushd $subdir
     # Build the binaries
     make \
         -j 4 \
-        DYN_LIBS="-lz -lpthread -lbz2 -llzma -lcurl -lhts -ldeflate" \
-        CXX="$CXX -std=c++11" \
-        CXXFLAG="$CXXFLAGS ${PREFIX}" \
+        DYN_LIBS="-lz -lpthread -lbz2 -llzma -lcurl -lhts -ldeflate -lm" \
+        CXX="$CXX -std=c++17" \
+        CXXFLAG="$CXXFLAGS ${PREFIX} -D__COMMIT_ID__=\"${COMMIT_VERS}\" -D__COMMIT_DATE__=\"${COMMIT_DATE}\" -Wno-ignored-attributes -O3 -mavx2 -mfma" \
```

Comment:
I am grasping at straws here, how about this?
```suggestion
        CXXFLAG="$CXXFLAGS ${PREFIX} -D__COMMIT_ID__='\"${COMMIT_VERS}\"' -D__COMMIT_DATE__='\"${COMMIT_DATE}\"' -Wno-ignored-attributes -O3 -mavx2 -mfma" \
```

Reply:
Looking at the output, it seems to work.
We just need to find a way for the version. GIT_FULL_HASH, does not seems to work.
Should we harcode it ?